File: spec/itemSpec.js
let item = require("../src/itemClass.js");

describe("Item", function() {

  describe("Assigning values to the item", function() {
    it("should return the correct name, quality and value of the item", function() {
      item = new item.testItem("Cherries", 10, 10);
      expect(item.name).toEqual("Cherries");
      expect(item.sellIn).toEqual(10);
      expect(item.quality).toEqual(10);

    });
  });

});

File: spec/item_spec.rb
require "item"
require_relative "spec_helper"
require "pry"

describe "item" do

  before(:each) do
     @item = Item.new("cherry", 10, 10)
  end
  it "returns the name, quality and sell in date of the item" do
    #item = Item.new("cherry", 10, 10)
    expect(@item.name).to eq("cherry")
    expect(@item.sell_in).to eq(10)
    expect(@item.quality).to eq(10)
  end

  describe "#to_s" do
    it "returns in string format the name, sell in and quality of the item" do
      expect(@item.to_s).to eq("cherry, 10, 10")
    end
  end
end